dali/python/nvidia/dali/sysconfig.py

```diff
@@ -33,6 +33,17 @@ def get_lib_dir():
     import nvidia.dali as dali
     return os.path.dirname(dali.__file__)
 
+def get_include_flags():
+    """Get the include flags for custom operators
+
+    Returns:
+        The compilation flags
+    """
+    import nvidia.dali.backend as b
+    flags = []
+    flags.append('-I%s' % get_include_dir())
```

Comment:
The strange thing here is that there's just one include dir...

Reply:
just following the convention of the other functions. Also, we might want to add more of them in the future